stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -353,7 +353,7 @@
         <item name="streamUiMessageTextStyleThreadSeparator">normal</item>
 
         <!-- Message Reply background color -->
-        <item name="streamUiMessageReplyBackgroundColorMine">@color/stream_ui_bars_background</item>
+        <item name="streamUiMessageReplyBackgroundColorMine">@color/stream_ui_grey_gainsboro</item>
```

Comment:
Any reason for the color change?

Reply:
Take a look at the comment above with the image reply.